{{cookiecutter.repo_name}}/Makefile

```diff
@@ -43,7 +52,26 @@ quality:
 	pep8 --config=.pep8 {{cookiecutter.repo_name}} *.py
 	pylint --rcfile=pylintrc {{cookiecutter.repo_name}} *.py
 
-validate: test quality
+serve:
```

Comment:
We're moving to docker for devstack (and local development), which is what the port is being used for. See the `start-devstack` target. Do we need a target for a local run-server as well?